app/Http/Controllers/BeatmapsController.php

```diff
@@ -87,7 +87,7 @@ public function attributes($id)
         }
 
         if (isset($params['mods'])) {
-            if (ctype_digit($params['mods'])) {
+            if (is_numeric($params['mods'])) {
```

Comment:
`1.1` is numeric too (but then gets truncated by the cast) 🤔

Reply:
weird people deserve weird response. I don't think the function throws error for extraneous bits either so negatives and invalid bits are still accepted regardless